apps/billing/migrations/0014_auto_20161128_2324.py

```diff
@@ -12,6 +12,10 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
+        migrations.RemoveField(
```

Comment:
Why not just a [RenameField]( https://docs.djangoproject.com/en/1.10/ref/migration-operations/#renamefield)?